src/components/components/Card.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4">
+        <div class="mdc-card">
+            <div class="mdc-card__media">
+                <img :src=data.image width='100%' height='240'>
```

Comment:
Not sure `100%` is allowed as a value, only pixels IIRC.
Also, take a look at https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit